Doc/library/sys.rst

```diff
@@ -1292,6 +1292,66 @@ always available.
       The coroutine wrapper functionality has been deprecated, and
       will be removed in 3.8. See :issue:`32591` for details.
 
+.. function:: set_unawaited_coroutine_tracking_enabled(enabled)
+
+   Enables or disables "unawaited coroutine tracking mode". This is a
```

Comment:
Looks like the `sys` page uses the command form, so I believe this should be `Enable or disable`.  Same with the other new function descriptions.